cmd/zed/manage/ztests/once.yaml

```
@@ -0,0 +1,15 @@
+script: |
+  export ZED_LAKE=test1
+  zed init -q
+  zed create -S 10KB -q test1
```

```suggestion
  zed create -q test1
```

In the past I remember we had to use these small object sizes in order to see compaction in action because the heuristic was (something like) that there had to be an accumulation of a certain amount of uncompacted data relative to the object size. Based on the text description attached to your PR and my black box testing of this branch at commit 2cd633a it appears that even with the default object size compaction will be triggered, and so if that's all working as intended, then it seems we might as well test with default object size.

---

I'm going to add one more test that makes sure a non-overlapping object that is at least half the size of pool threshold is left alone.

---

Oh wait nevermind, I'll create another pool for the test. Will change to what you suggest.